droidmediacodec.cpp

```diff
@@ -17,14 +17,26 @@
  */
 
 #include <media/stagefright/OMXClient.h>
+
+// MediaCodecSource should be used instead of OMXCodec from Android 5
+#if ANDROID_MAJOR >= 5
+#include <media/stagefright/MediaBuffer.h>
+#include <media/stagefright/MediaCodecList.h>
+#include <media/stagefright/MediaCodecSource.h>
+#include <media/stagefright/MediaCodec.h>
+#include <media/openmax/OMX_Video.h>
+#include <media/openmax/OMX_Audio.h>
+#include <media/stagefright/ACodec.h>
+#include <media/stagefright/foundation/ALooper.h>
+#include <media/stagefright/foundation/AMessage.h>
+#endif
+
 #if ANDROID_MAJOR < 7
```

Comment:
should this be in #else of the above instead? This is only for ANDROID_MAJOR < 5 now right?

Reply:
No that's for 6 and below. There are constants in the flags that only got moved in 7 (kSoftwareCodecsOnly & kHardwareCodecsOnly).